packages/coinstac-docker-manager/src/index.js

```diff
@@ -570,20 +570,24 @@ const removeImage = (imageId) => {
  * @return {Promise}              A promise that resovles on success
  */
 const stopService = (serviceId, serviceUserId) => {
-  if (services[serviceId].users.indexOf(serviceUserId) > -1) {
+  if (services[serviceId] && services[serviceId].users.indexOf(serviceUserId) > -1) {
```

Comment:
For legibility sake I would put this `services[serviceId]` into a variable. Also I would do this on the beginning of the function `if (!services[serviceId]) return Promise.resolve()`, just to keep to the pattern of returning soon if you find an error.

Reply:
👍